### DISCUSSION TITLE
Questions Regarding the Validation of Potential of Mean Force (PMF)

### DISCUSSION THREAD

**Discussion by zhangtrzhangtr** [ricalessandri/Martini3-small-molecules#58] [Q&A]
I encountered an issue while calculating the Potential of Mean Force (PMF) for a Martini3 model I constructed. Following the approach described in your previous article, I performed umbrella sampling by controlling the distance between the center of mass of two molecules to obtain force information, and then used gmx wham for analysis and curve plotting. However, my molecular model is relatively large compared to molecules such as benzene, naphthalene, or thiophene, which were discussed in the previous article. My model consists of several components, and the resulting PMF curve differs significantly from the trends I expected.

Below is the .mdp file I used to set up the pulling in my simulation:
; Pull type: no, umbrella, constraint or constant_force
pull                     = yes

; Pull geometry: distance, direction, cylinder or position
pull-coord1-type            = umbrella
pull-coord1-geometry        = distance
; Select components for the pull vector. default: Y Y Y
pull-coord1-dim                 = Y Y Y
; Switch from r1 to r0 in case of dynamic reaction force
pull_constr_tol          = 1e-06
pull-coord1-start               = no
pull_nstxout             = 100
pull_nstfout             = 100
; Number of pull groups
pull_ngroups             = 2
; Group name, weight (default all 1), vector, init, rate (nm/ps), kJ/(mol*nm^2)
pull-group1-name  =  Y61
pull-group2-name  =  Y62
pull-coord1-groups              = 1 2
pull-coord1-init               = 0.48
pull-coord1-rate               = 0
pull-coord1-k                  = 800
pull-coord1-kB                 = 800

pull-pbc-ref-prev-step-com = yes
pull-group1-pbcatom      = 1
pull-group2-pbcatom      = 56
For the pull groups Y61 and Y62, I specified three atoms from my model as a pull group using an .ndx file. These atoms, Y61 and Y62, come from two different molecules I built. I placed two of my constructed molecular models in the system, filled it with solvent, and positioned the initial configuration. Below is the picture showing my Martini3 molecular model and the selected atoms from each molecule that were used as the pull groups, as well as the initial distance configuration.

I would like to ask where the potential issue might be. Specifically, is validating the PMF for Martini3 models of large molecules with more components necessary? Additionally, would it be feasible to break down the large molecule into smaller parts for PMF validation, comparing the results to those from all-atom molecular dynamics simulations? I would greatly appreciate your guidance and assistance in resolving this, and I thank you sincerely for your time.
![supportinginformation](https://github.com/user-attachments/assets/d1eefd9f-9125-493a-96ca-1f352f1ed92b)

**Comment by ricalessandri** [comment]
@zhangtrzhangtr Your pulling settings look fine, assuming that the example file is only for the window at a distance of 0.48 (pull-coord1-init = 0.48), and that you will sample along the CV with each simulation window having a different pull-coord1-init value. I’ve used a force constant of 1500 kJ mol−1 nm−2, while you are using 800, but that shouldn't make a significant difference.

Comprehensive sampling of the PMF of dimerization for such a large molecule based solely on the center-of-mass (com-com) or center of geometry (cog-cog) distance might be challenging. Validating fragments of the models by comparing them to all-atom simulations sounds like a great approach, and I would recommend it. Once the AA versus CG comparison for the fragments looks satisfactory, you could assume that your Y61 model will be reliable as well, and you wouldn’t need to perform an AA PMF of dimerization on the full molecule (which is quite challenging to converge).

However, keep in mind that you will need to ensure the convergence of the CG PMF for Y61. A good first step is to run progressively longer simulations at each window and perform WHAM analysis to check for changes in the PMF. This doesn’t guarantee exhaustive sampling, though, since some windows might get stuck in local minima, which umbrella sampling wouldn’t resolve. Therefore, another important check is to visualize or analyze whether each window (especially those at lower distances) is adequately exploring the conformational space. I hope these initial thoughts help you move forward.
